src/scheduler/constant.hpp

```diff
@@ -439,9 +440,9 @@ enum prime_time
 {
 	NON_PRIME = 0,
 	PRIME = 1,
-	ALL,
-	NONE,
-	HIGH_PRIME
+	HIGH_PRIME,
+	PT_ALL,
+	PT_NONE
```

Comment:
doing this will cause `struct t prime[HIGH_DAY][HIGH_PRIME]` to be much bigger than it needs to be, since PT_ALL and PT_NONE would be counted in in the array.

Reply:
You make a good point.  How about we just hard code 2 instead of HIGH_PRIME and get rid of HIGH_PRIME.